styles.css

```diff
@@ -1,3 +1,345 @@
+}
+
+.header {
+  font-style: normal;
+  font-weight: 400;
+  font-size: 32px;
+  line-height: 44px;
+  color: #091e42;
+  margin-top: 10px;
+  margin-bottom: 0;
+}
+
+.main-layout2 {
+  padding: 18px;
+}
+
+.main-layout2 p {
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+}
+
+.main-layout2 button {
+  padding: 12px;
+  background: #ff6b00;
+  margin: 7px 0;
+  border: 0 solid transparent;
+  color: white;
+}
+
+.about-section {
+  background: #1c1a19;
+  padding: 18px;
+  display: flex;
+  flex-direction: column;
+  color: #fff;
+}
+
+.intro h4 {
+  font-style: normal;
+  font-weight: 400;
+  font-size: 72px;
+  line-height: 88px;
+  letter-spacing: 0.37px;
+  color: #ff6b00;
+  margin-top: 30px;
+  margin-bottom: 0;
+}
+
+.intro p {
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #f4f5f7;
+}
+
+.intro button {
+  padding: 12px;
+  background: #ff6b00;
+  border: 0 solid transparent;
+  color: white;
+}
+
+.langs h3 {
+  font-style: normal;
+  font-weight: 700;
+  font-size: 32px;
+  line-height: 44px;
+  display: flex;
+  align-items: center;
+  color: #fff;
+  margin: 20px 0;
+}
+
+.bg-about {
+  background-image: url('./Assets/Groupx.png');
+  background-repeat: no-repeat;
+  background-position: right -0 bottom 10px;
+  height: 200px;
+  background-size: 100px 200px;
+  background-size: 200px 100px;
+}
+
+.langs h6 {
+  font-style: normal;
+  font-weight: 400;
+  font-size: 22px;
+  line-height: 43px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  color: #fff;
+  margin: 0;
+}
+
+.langs {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.langs svg {
+  margin: 0 10px;
+}
+
+.contact {
+  padding: 18px;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 32px;
+  line-height: 44px;
+  text-align: center;
+  color: #172b4d;
+}
+
+.contact button {
+  padding: 12px;
+  background: #ff6b00;
+  margin: 7px 0;
+  border: 0 solid transparent;
+  color: white;
+}
+
+/*
+.contact {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+} */
+
+.form-info {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: space-between;
+  width: 80%;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+.form-info input {
+  margin: 10px;
+  border-width: 0 0 2px 0;
+  border-style: solid;
+  border-color: lightgray;
+  height: 30px;
+  width: 100%;
+}
+
+.form-info textarea {
+  margin: 10px;
+  border-width: 0 0 2px 0;
+  border-style: solid;
+  border-color: lightgray;
+  height: 70px;
+  width: 100%;
+}
+
+.form {
+  width: 80%;
+  display: inline-block;
+}
+
+.framwork-links span {
+  font-style: normal;
+  font-weight: 500;
+  font-size: 15px;
+  color: #091e42;
+  border: 1px solid lightgray;
+  padding: 10px;
+}
+
+.reads {
+  padding: 28px 18px;
+  color: white;
+  width: 70%;
+  margin-left: auto;
+  margin-right: auto;
+  margin-bottom: 10px;
+}
```

Comment:
- [ ] Please 🙏🏽 use display flexbox and grid in the work section, for the best practices for the HTML & CSS.

> In order to lay out the elements in "Works" section you should use CSS Grid and Flexbox combined (it is mandatory that you use both). Use CSS Grid in the main container that contains all of the cards and use Flexbox to lay out an individual card.